cisco_umbrella_dns/assets/dashboards/cisco_umbrella_dns_dns_traffic.json

```
@@ -0,0 +1,1171 @@
+{
+    "title": "Cisco Umbrella DNS - DNS Traffic",
+    "description": "-   This dashboard provides detailed information on DNS traffic.",
```

Can this be more descriptive? 

```suggestion
    "description": "This dashboard provides detailed information on DNS traffic.",
```

---

Updated descriptions of both dashboards